.pre-commit-config.yaml

```diff
@@ -80,22 +87,15 @@ repos:
 
   # Python hooks
   - repo: https://github.com/PyCQA/bandit
-    rev: 1.7.1
+    rev: 1.7.4
     hooks:
       - id: bandit
         name: bandit (tests tree)
         files: tests
         args:
           - --config=.bandit.yml
-  # Run bandit everything but tests directory
-  - repo: https://github.com/PyCQA/bandit
-    rev: 1.7.0
-    hooks:
-      - id: bandit
-        name: bandit (everything else)
-        exclude: tests
-  - repo: https://github.com/python/black
-    rev: 21.12b0
```

Comment:
Why was this block removed?

Reply:
resolved with [08acc63](https://github.com/cisagov/domain-manager-ui/pull/247/commits/08acc63eebd75bfe77a1b1058cdad6e544ba5677)